.github/workflows/make-ci.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: Make CI
+
+'on':
+  workflow_call:
+    inputs:
+      timeout_minutes:
+        description: The maximum time (in minutes) for a job to run.
+        default: 10
+        required: false
+        type: number
+
+jobs:
+  make-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22.x"
+      - name: Install prettier
+        run: yarn add prettier@3
+      - name: Put node executables in the system PATH
+        run: echo -n "$(yarn bin)" >> $GITHUB_PATH
```

Comment:
Did you look into using one of the pre-existing Prettier actions from the Marketplace?

https://github.com/marketplace?query=prettier

I'm not sure how good any of them are, but if any would work it would be less future maintenance we would need to do for this.

Reply:
using an action here fundamentally breaks what we're trying to do here, which is encapsulate the CI entirely within a single makefile command